src/views/addManager/addManager.js

```diff
@@ -0,0 +1,280 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { Form, Field, Formik } from "formik";
+import * as Yup from "yup";
+import * as axios from "axios";
+import { AddProperty } from '../addProperty/addProperty';
+import { SearchPanel, SearchPanelVariant } from "react-search-panel";
+import UserContext from "../../UserContext";
+import Button from "../../components/Button";
+import Modal from '../../components/Modal';
+import Toast from '../../utils/toast';
+import useMountEffect from '../../utils/useMountEffect';
+import './addManager.scss';
+import RoleEnum from '../../Enums/RoleEnum';
+
+const validationSchema = Yup.object().shape({
+  firstName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a First Name"),
+  lastName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a Last Name"),
+  phone: Yup.string()
```

Comment:
Is there a reason the validation does length 5 - 20?  Accounting for non-standard/international #?